src/User32.Desktop/User32.cs

```diff
@@ -621,10 +674,478 @@ public static partial class User32
+        /// </param>
+        /// <returns>
+        /// If the function succeeds, the return value is a handle to the newly created desktop.
+        /// If the specified desktop already exists, the function succeeds and returns a handle to the existing desktop.
+        /// If the function fails, the return value is an invalid handle.
+        /// </returns>
+        /// <remarks>
+        /// <para>
+        /// If the dwDesiredAccess parameter specifies the READ_CONTROL, WRITE_DAC, or WRITE_OWNER standard access rights, you must also request the DESKTOP_READOBJECTS and DESKTOP_WRITEOBJECTS access rights.
+        /// </para>
+        /// <para>
+        /// The number of desktops that can be created is limited by the size of the system desktop heap, which is 48 MB. Desktop objects use the heap to store resources. You can increase the number of desktops that can be created by reducing the default heap reserved for each desktop in the interactive window station. This value is specified in the SharedSection substring of the following registry value: HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\SubSystems\Windows.
+        /// </para>
+        /// </remarks>
+        [DllImport(nameof(User32), CharSet = CharSet.Unicode, ExactSpelling = true, SetLastError = true)]
+        public static unsafe extern SafeDesktopHandle CreateDesktop(
+            string lpszDesktop,
+            string lpszDevice,
+            IntPtr pDevmode,
+            DesktopCreationFlags dwFlags,
+            DESKTOP_ACCESS_MASK dwDesiredAccess,
+            [Friendly(FriendlyFlags.In | FriendlyFlags.Optional)] Kernel32.SECURITY_ATTRIBUTES* lpsa);
+
+        /// <summary>
+        /// Retrieves a handle to the desktop assigned to the specified thread.
+        /// </summary>
+        /// <param name="dwThreadId">The thread identifier. The <see cref="Kernel32.GetCurrentThreadId"/> and CreateProcess functions return thread identifiers.</param>
+        /// <returns>
+        /// If the function succeeds, the return value is a handle to the desktop associated with the specified thread.
+        /// You do not need to call the <see cref="CloseDesktop"/> function to close the returned handle.
+        /// </returns>
+        /// <remarks>
+        /// <para>
+        /// The system associates a desktop with a thread when that thread is created. A thread can use the SetThreadDesktop function to change its desktop. The desktop associated with a thread must be on the window station associated with the thread's process.
+        /// </para>
+        /// The calling process can use the returned handle in calls to the <see cref="GetUserObjectInformation(IntPtr, ObjectInformationType, void*, uint, uint*)"/>, GetUserObjectSecurity, SetUserObjectInformation, and SetUserObjectSecurity functions.
+        /// <para>
+        /// </para>
```

Comment:
Hahaha. I guess it was meant to have the text that is just above it and I missed the placement completely.